File: src/main/resources/static/js/unit.js
$("#addNew").click(function () {
    document.getElementById("pop").style.display = 'block';
    document.getElementById("pop").scrollIntoView()
})

$("#cancel").click(function () {
    document.body.style.backgroundColor = "#ffffff"
    document.getElementById("pop").style.display = 'none';
})
$("#confirm").click(function () {
    var adata = {
        "unitId": $("input[ name='unitId' ]").val(),
        "name": $("input[ name='name' ]").val(),
        "nature": $("input[ name='nature' ]").val(),
        "location": $("input[ name='location' ]").val(),
        "remarks": $("input[ name='remarks' ]").val(),
        "phone": $("input[ name='phone' ]").val()
    }
    var data = JSON.stringify(adata);
    console.log(data)
    $.ajax({
        type: "POST",
        data: data,
        url: "/admin/unit/save",
        headers: {"Content-Type": "application/json;charset=utf-8"},
        success: function (message) {
            alert(message)
        },
        error: function () {
            alert("保存失败")
        }
    })
})

File: src/main/java/com/library/entities/Unit.java
package com.library.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class Unit implements Serializable {
    private String unitId;//单位id
    private String name;//单位名称
    private Integer nature;//性质
    private String location;
    private String remarks;//单位备注
    private String phone;
}
